### ISSUE TITLE
Setup USDS staging environment

### ISSUE THREAD

**Issue by mikepif** [newjersey/dhs-oneapp#48]
We need to setup a staging environment at USDS for internal testing.

### Cloud.gov account
- [x] Assemble list of required services we're going to use on Cloud.gov
- [x] Reach out to Lisa about using USDS's Cloud.gov account
- [x] Wait for Cloud.gov response on raising disk quota
- [x] Deploy out application to cloud.gov after Oracle database is opened up (blocked)
- [x] Communicate to Lisa the resource usage for cloud.gov
- [ ] Migrate app to USDS cloud.gov account (Waiting on available resources in NJ database) (#181)

### Database
- [x] Prepare database for Cloud.gov deploy
- [x] Wait for NJ to open up Oracle database

### Back-end
- [x] Prepare back-end for Cloud.gov deploy (blocked)
- [x] Connect back-end to database
- [x] Deploy out the back-end to Cloud.gov

### Front-end
- [x] Prepare front-end for Cloud.gov deploy (blocked)
- [x] Deploy out the front-end to Cloud.gov
- [x] Connect front-end to back-end

### CD
- [x] Setup automatic deployments when code is merged into `main`

**Comment by Samara-Strauss** [comment]
Currently blocked on discussion with Tom to determine how to get access to NJ database or AWS environment.

**Comment by Samara-Strauss** [comment]
To unblock: Schedule meeting with Tom.

Leah has emailed; waiting to hear back.

**Comment by ronaldheft-usds** [comment]
Reached out to NJ about the firewall rules for the database that are still in progress.

**Comment by Samara-Strauss** [comment]
NJ got back to us and says firewall rules are in place, but Ron needs more information on how we can access it from Cloud.gov.

**Comment by Samara-Strauss** [comment]
This is effectively blocked until we have access to the Oracle database.

**Comment by ronaldheft-usds** [comment]
Updated status. NJ opened up the database, but something is misconfigured. Reached out this morning to them with technical details for their assistance on resolving. There are a few other items I can continue with before getting blocked again.

**Comment by Samara-Strauss** [comment]
Should be closed this week assuming the connectivity issues with NJ are resolved.

**Comment by ronaldheft-usds** [comment]
Everything is now done but migrating to USDS account. Created #181 to track that work.

**Comment by Samara-Strauss** [comment]
Awesome! Congratulations! Closing this out.
